README.md

```diff
@@ -1 +1,38 @@
-# jenkins-k8s-operator
\ No newline at end of file
+# jenkins-k8s-operator
+
+**This is a WIP project.**
```

Comment:
Should we remove this to avoid the risk or forgetting to remove it?

Reply:
Done and done! Thanks for the reviews!